core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaUpdateRecipesTranslator.java

```diff
@@ -94,17 +94,27 @@ public class JavaUpdateRecipesTranslator extends PacketTranslator<ClientboundUpd
             "minecraft:netherite_boots"
     );
 
+    /**
+     * Fixes https://github.com/GeyserMC/Geyser/issues/3784 by using item tags where applicable instead of group IDs
+     * Item Tags allow mixing ingredients, and theoretically, adding item tags to custom items should also include them.
+     */
+    private static final Map<String, String> RECIPE_TAGS = Map.of(
+    "minecraft:wood", "minecraft:logs",
```

Comment:
Weird formatting here? Use the same formatting as above.

Reply:
Oh yeah, mb. https://github.com/GeyserMC/Geyser/pull/4016/commits/5e5fbb7959798ec4fe22c947a8da4f8d173921ed fixes it